### PR TITLE
Add filer to docker-compose and entrypoint and build from local Dockerfile

### PR DESCRIPTION
**Pull request by hans-strudle**
I was playing around with seaweedfs in docker, and added a filer option to the `entrypoint.sh` as well as in the `docker-compose.yml`, to easily bring up a filer alongside the master & volume. This includes adding in the `filer.toml` which I have set to use the memory store.

I also would make the argument that this docker-compose should be building from the local Dockerfile, hence the change to using `build: .` instead of `image: chrislusf/seaweedfs` in the `docker-compose.yml`.

Also sorry for the dirty commit log with the odd usernames :)

### CODE REVIEW

File: docker/filer.toml [hunk 1]
```diff
@@ -0,0 +1,3 @@
+[memory]
+enabled = true
```
Comment (chrislusf) [inline docker/filer.toml]: how about default to leveldb? 

If just in memory, the directories will be gone while the files are still stored on volume servers.

File: docker/docker-compose.yml [hunk 2]
```diff
@@ -2,7 +2,7 @@ version: '2'
 
 services:
   master:
-    image: chrislusf/seaweedfs
+    build: .
```
Comment (chrislusf) [inline docker/docker-compose.yml]: maybe put a comment here to show the different options, either build local or use the image. same for other 2 places.

**Comment by chrislusf** [comment]
Thanks for the PR!

Added a few comments. Do not worry about Travis CI build error.

File: docker/filer.toml (hunk 1 above)
Comment (hans-strudle) [reply docker/filer.toml]: Great point, I assumed leveldb required some sort of extra setup, but just tested and it does preserve the information on a container restart. Although state will be lost if the container is rebuilt without the use of a volume of course.

File: docker/docker-compose.yml (hunk 2 above)
Comment (hans-strudle) [reply docker/docker-compose.yml]: Added comments :)

**Pull request merged**